src/main/java/com/volasoftware/tinder/service/UserService.java

```diff
@@ -135,4 +135,19 @@ public UserProfileDto getCurrentUserProfile() {
 
         return new UserProfileDto(user.getFirstName(), user.getLastName(), user.getEmail(), user.getGender());
     }
+
+    public UserProfileDto editUserProfile(@RequestBody UserProfileDto userProfileDto){
```

Comment:
Missing a space between ) and {. Just after you write some code use Ctrl+Alt+L (auto formatting) for it. Try to create this habit